queries/ralph/highlights.scm

```diff
@@ -0,0 +1,135 @@
+(type_identifier) @type
+
+[
+  (primitive_type)
+  (alph_token_id)
+] @type.builtin
+
+[
+  (var_declaration)
+  (struct_field)
+] @variable
+
+(arg_def
+  (identifier) @variable)
+
+(struct_constructor_field
+  (identifier) @variable)
+
+[
+  (hex_num)
+  (integer)
+  (typed_num)
+] @number
+
+[
+  (bool)
+  (bytes)
+  (address)
+] @constant.builtin
+
+(func_id) @function
+
+[
+  (mutable)
+  (annotation)
+] @keyword
+
+(annotation
+  (identifier) @keyword)
+
+(annotation_field
+  (identifier) @variable)
+
+(annotation_field
+  (identifier) @variable)
+
+(map_def
+  (identifier) @variable)
+
+"fn" @keyword.function
+
+[
+  "Contract"
+  "TxScript"
+  "Interface"
+  "AssetScript"
+  "struct"
+  "enum"
+  "event"
+  "mapping"
+] @keyword.type
+
+[
+  "const"
```

Comment:
I think this one can be just `@keyword` (Sorry if this was confusing: we usually highlight `"const"` as a modifier in the case of something like `pub const function myfunc() ...`, but not in variable declarations like `const myvar = 3`. I should mention this in the docs!)